test/sql/types/enum/test_enum.test

```diff
@@ -39,8 +42,8 @@ statement error
 CREATE TYPE bla AS ENUM ('sad',NULL);
 
 # We don't accept repeated elements?
-statement error
-CREATE TYPE bla AS ENUM ('sad','sad');
+# statement error
+# CREATE TYPE bla AS ENUM ('sad','sad');
```

Comment:
This test should fail - but indeed it appears the test was not correctly checking this and it currently succeeds. If you could fix it in this PR as well that could be great, otherwise we can open a separate issue for it.